lib/chef/knife/bootstrap_windows_base.rb

```diff
@@ -319,6 +323,28 @@ def bootstrap(proto=nil)
         end
 
         wait_for_remote_response( config[:auth_timeout].to_i )
+
+        # We allow the user to specify the desired architecture of Chef to install or we default
+        # to whatever the target system is.  We assume that we are only bootstrapping 1 node at a time
+        # so we don't need to worry about multipe responses from this command.
+        session_results = relay_winrm_command("echo %PROCESSOR_ARCHITECTURE%")
+        # TODO is this valid error checking?
```

Comment:
@mwrock What do you think about the error checking I'm doing here?  I'm not sure what kind of responses I will get from `relay_winrm_command` so If its at all different from what I expect I raise an error.